### PR TITLE
Add readme in distributed_training

### PR DESCRIPTION
**Pull request by KumoLiu**
Add readme in distributed_training

### Checks

- [ ] Avoid including large-size files in the PR.
- [ ] Clean up long text outputs from code cells in the notebook.
- [ ] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [ ] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [ ] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

File: acceleration/distributed_training/distributed_training.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Data Preparation
+
+Users need to download the dataset Task01_BrainTumour from the MICCAI challenge [Medical Segmentation Decathlon](http://medicaldecathlon.com/) for the following examples.
```
Comment (mingxin-zheng) [inline acceleration/distributed_training/distributed_training.md:3]: Hi @KumoLiu , thanks for the PR.

Should we have an example download snippet here for the dataset, for example:
```bash
# Define the directory where to download and extract the dataset
root_dir="/path/to/your/directory"  # Change this to your desired directory

# Download and extract in one line if the directory doesn't exist
[ ! -d "${root_dir}/Task01_BrainTumour" ] && mkdir -p "${root_dir}/Task01_BrainTumour" && wget -qO- "https://msd-for-monai.s3-us-west-2.amazonaws.com/Task01_BrainTumour.tar" | tar -xv -C "${root_dir}"
```

File: acceleration/distributed_training/distributed_training.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Multi-GPU Training
+
+Users can set your `NUM_GPUS_PER_NODE`, `NUM_NODES`, `INDEX_CURRENT_NODE`, as well as `DIR_OF_TESTDATA` for the directory of the test dataset.
```
Comment (mingxin-zheng) [inline acceleration/distributed_training/distributed_training.md:7]: Does the test dataset mean the entire dataset (Task01_BrainTumour) to the imagesTs?

Comment (KumoLiu) [reply acceleration/distributed_training/distributed_training.md:7]: I think the `DIR_OF_TESTDATA` here just means that the data used for this tutorial. Not real "testing data".

Comment (KumoLiu) [reply acceleration/distributed_training/distributed_training.md]: Updated to `DIR_OF_DATA`.

File: acceleration/distributed_training/distributed_training.md (hunk 1 above)
Comment (KumoLiu) [reply acceleration/distributed_training/distributed_training.md:3]: Updated it in the latest commit, thanks.

**Review by mingxin-zheng — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**